src/languages/yaml.js

```diff
@@ -45,8 +45,9 @@ export default function(hljs) {
   };
 
   return {
+    name: 'YAML',
     case_insensitive: true,
-    aliases: ['yml', 'YAML', 'yaml'],
```

Comment:
Can we add this back?  It’s not really part of the name change and it’s not consistent either. We need to look into how we handle case with aliases and then approach this problem across the board.